data-reconciliation-app/Makefile

```diff
@@ -58,10 +58,10 @@ unit-test: ## 🔨 Run the Application unit-test
 # To build and start a ccf network and automatically deploy your application on it.
 demo: ## 🎬 Demo the Data Reconciliation Application
 	@echo -e "\e[34m$@\e[0m" || true
-	@./demo-app/scripts/01-data_ingest_demo.sh
-	@./demo-app/scripts/02-report_change_demo.sh
+	@./test/01-data_ingest_demo.sh
+	@./test/02-report_change_demo.sh
 
 demo-code-change: build ## 🎬 Demo the Data Reconciliation Application Code Change
 	@echo -e "\e[34m$@\e[0m" || true
 	$(call check_defined, CCF_NAME)
-	@./demo-app/scripts/03-code_change_demo.sh --address "${CCF_NAME}.confidential-ledger.azure.com"
\ No newline at end of file
+	@./test/03-code_change_demo.sh --address "${CCF_NAME}.confidential-ledger.azure.com"
```

Comment:
nit pick and not related to this issue, but as you touched this part:

this seems like a dangling target, as it is only related to mCCF and expect some previous commands to be executed before. IMHO this  needs to be adjusted prior to hand-off.
Do we have a ticket for this already?

Reply:
I think this will be picked up by another ticket. This PR is effectively getting the demo-app folder into the test folder so that we have a single place to work from.